classes/Organisations.js

```diff
@@ -24,24 +24,27 @@ module.exports = (tenant /* : Tenant */) =>
     }
 
     uploadAsset(
-      assetData /* :string */,
+      assetData /* :mixed */,
       assetFileName /*:string */,
+      assetContentType /* :?string */,
```

Comment:
Can we change this to an single parameter as an object, easy to get these in the wrong order in JS.

Reply:
IMO

```json
{
  "data": "",
  "fileName": "",
  "contentType": "",
}